admin/tabs/activity/tracking/ActivityModel.js

```diff
@@ -0,0 +1,227 @@
+        enableClear: true,
+        dimensions: [
+            {label: 'Date', value: 'day'},
+            {label: 'User', value: 'username'},
+            {label: 'Message', value: 'msg'},
+            {label: 'Category', value: 'category'},
+            {label: 'Device', value: 'device'},
+            {label: 'Browser', value: 'browser'},
+            {label: 'User Agent', value: 'userAgent'}
+        ],
+        initialValue: ['day', 'username', 'msg']
+    });
+
+    @managed
+    cube = new Cube({
+        fields: [
+            {name: 'day', isDimension: true, aggregator: new RangeAggregator()},
+            {name: 'username', isDimension: true, aggregator: 'UNIQUE'},
+            {name: 'msg', isDimension: true, aggregator: 'UNIQUE'},
+            {name: 'category', isDimension: true, aggregator: 'UNIQUE'},
+            {name: 'device', isDimension: true, aggregator: 'UNIQUE'},
+            {name: 'browser', isDimension: true, aggregator: 'UNIQUE'},
+            {name: 'userAgent', isDimension: true, aggregator: 'UNIQUE'},
+
+            {name: 'elapsed', aggregator: 'AVG'},
+            {name: 'impersonating'},
+            {name: 'dateCreated'},
+            {name: 'data'},
+            {name: 'count', aggregator: new ChildCountAggregator()},
+            {name: 'entryCount', aggregator: new LeafCountAggregator()} // Used for charting, not displayed in grid,
+        ]
+    })
+
+    @managed
+    gridModel = new GridModel({
+        treeMode: true,
+        stateModel: 'xhActivityGrid',
+        enableColChooser: true,
+        enableExport: true,
+        exportOptions: {filename: () => `Activity ${fmtDate(this.startDate)} to ${fmtDate(this.endDate)}`},
+        emptyText: 'No activity reported...',
+        sortBy: 'dateCreated|desc',
+        columns: [
+            {
+                field: 'cubeLabel',
+                headerName: 'Track',
+                width: 160,
+                isTreeColumn: true,
+                renderer: (v, params) => params.record.raw.cubeDimension === 'day' ? fmtDate(v) : v
+            },
+            {field: 'username', ...usernameCol},
+            {field: 'day', width: 200, align: 'right', renderer: this.dateRangeRenderer},
+            {field: 'category', width: 100},
+            {field: 'device', width: 100},
+            {field: 'browser', width: 100},
+            {field: 'userAgent', width: 100, hidden: true},
+            {field: 'impersonating', width: 140},
+            {
+                field: 'elapsed',
+                headerName: 'Avg Elapsed (ms)',
+                width: 130,
+                align: 'right',
+                renderer: numberRenderer({formatConfig: {thousandSeparated: false, mantissa: 0}})
+            },
+            {field: 'msg', headerName: 'Message', flex: true, minWidth: 120},
+            {field: 'data', width: 70},
+            {field: 'count', width: 70, align: 'right', renderer: numberRenderer({formatConfig: {thousandSeparated: false, mantissa: 0}})},
```

Comment:
this is weird -- let's just use the standard hoist formatting and wait for a fix to 

https://github.com/xh/hoist-react/issues/1931.   Can give this code snippet as and example in that ticket.